app/models/chaser_sent.rb

```diff
@@ -3,8 +3,12 @@ class ChaserSent < ApplicationRecord
 
   enum chaser_type: {
     reference_request: 'reference_request',
+    referee_reference_request: 'referee_reference_request',
+    candidate_reference_request: 'candidate_reference_request',
     reference_replacement: 'reference_replacement',
```

Comment:
Not a blocker but by being more consistent here with new enum entries we could predict the old enum type from the new enum type (ie the same `chaser_type` enum entry but without the `referee_` or `candidate_` prefix) this would mean you wouldn't need the `old_type` config entry and the enum would be explicit about who the recipient was.
ie. add `candidate_reference_replacement` and `referee_reference_nudge`

Then it's possible to do

```ruby
chaser_type = config[:chaser_type]
previous_chaser_type = chaser_type.sub(/^candidate_|^referee_/, '')
ChaserSent.where(chaser_type: [chaser_type, previous_chaser_type]
```
Another benefit is we make one database query for all the relevant `chased_ids`.

Reply:
Done! ♻️